src/frontend/delete-github-subscription.ts

```diff
@@ -66,13 +64,12 @@ export default async (req: Request, res: Response): Promise<void> => {
 			}
 
 			const subscription = await Subscription.getSingleInstallation(req.body.jiraHost, req.body.installationId);
-			const installation = await Installation.getForHost(req.body.jiraHost);
-			const action = ActionFromSubscription(subscription, installation);
-			action.type = ActionType.DESTROYED;
-			action.actionSource = ActionSource.WEB_CONSOLE;
-
+			if(!subscription) {
```

Comment:
It is a bit disturbing to see business logic changes in this PR, formatting changes affect many files and I am afraid I will miss something

Reply:
It wouldn't compile without optional chaining or a conditionals since there's no guarantee that a subscription is returned.  I could just leave it as optional chaining, but figured we'd want to log when this happens as it shouldn't.